contrib/ci/dependencies.xml

```diff
@@ -70,6 +70,25 @@
       <package variant="clang" />
     </distro>
   </dependency>
+  <dependency type="build" id="libjson-glib-dev">
+    <distro id="fedora">
+      <package>json-glib-devel</package>
+    </distro>
+    <distro id="debian">
+      <control>
+        <version>(>= 1.1.1)</version>
+      </control>
+      <package variant="x86_64" />
+      <package variant="s390x">libjson-glib-dev:s390x</package>
+      <package variant="i386" />
+    </distro>
+    <distro id="ubuntu">
+      <control>
+        <version>(>= 1.1.1)</version>
+      </control>
+      <package variant="clang" />
+    </distro>
+  </dependency>
```

Comment:
Isn't arch missing?  I'm surprised arch CI is letting it through, it should be needing this: https://www.archlinux.org/packages/extra/x86_64/json-glib/

Reply:
Fixed. I assume it's already in the base install, but it's better to be explicit.